shapeshift/web
# feat: upgrade walletconnect ui to align to multiple connection design

## Description

Upgrades the ui of the walletconnect dropdown to align to the multi-connection design. Does not implement multi-connections though - that will be in a follow-up.

Note that multiple address links appear as duplicates in the UI but point to different pages - this aligns to the design:
![image](https://github.com/shapeshift/web/assets/125113430/396cf869-4819-4457-a899-9abaa92c351f)

## Pull Request Type


## Issue (if applicable)

progresses #5236

## Risk

* Risk of broken walletconnect in app - connecting or disconnecting
* Risk of incorrect connection details

## Testing

* Check ability to connect+disconnect via walletconnect still works
* Check info displayed in ui is correct
* Check address links are correct

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)

Default state (before connecting walletconnect) - unchanged from prod
![image](https://github.com/shapeshift/web/assets/125113430/94115019-58c4-4429-b809-2dc117499a1f)

App connected - unchanged from prod
![image](https://github.com/shapeshift/web/assets/125113430/9476ff98-526f-4aa5-946e-6dc0afca1e47)

Dropdown opened, default state - aligned to new design
![image](https://github.com/shapeshift/web/assets/125113430/baf1e803-ffb7-44a9-a608-cdbc3a9c440c)

Dropdown opened, collapsed state - aligned to new design. This isn't useful until multi-connection ui is implemented, but placeholder functionality is there.
![image](https://github.com/shapeshift/web/assets/125113430/c617c52a-7fdc-40c9-9592-3d5a5be3cedd)
